.github/workflows/check.yml

```diff
@@ -1,20 +1,27 @@
 name: Check
 on:
   push:
-    branches: [ '**' ]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ '**' ]
+
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/datadog/images-rb/engines/ruby:3.2
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: bundle install
+      - run: bundle exec rake rubocop standard
```

Comment:
I think it would be clearer (for the person reading failures) to have rubocop and standard invoked separately.